gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -347,6 +347,11 @@ public class ConfigurationKeys {
    */
   public static final String WATERMARK_INTERVAL_VALUE_KEY = "watermark.interval.value";
 
+  /**
+   * DEFAULT LOOKBACK TIME KEY property
+   */
+  public static final String DEFAULT_COPY_LOOKBACK_TIME_KEY = "copy.lookbackTime";
```

Comment:
check the datapacks MP, this config is used to set another config (configX), which is used at various places in gobblin to find the lookback period. Please use that configX

Reply:
The property is ["gobblin.copy.recursive.lookback.time"](https://github.com/apache/gobblin/blob/master/gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/TimeAwareRecursiveCopyableDataset.java#L48), isn't it will be confusing to use other DatasetFinder config in other finder class ?

Reply:
not 100% clear here...

is the issue that:
* the same prop name is already in use elsewhere so we should avoid conflicting? 
* that a config w/ similar semantics already exists, and we might intentionally borrow it here for uniformity?
* or something else...?

Reply:
in other templates copy.lookbackTime is assigned to ["gobblin.copy.recursive.lookback.time"](https://github.com/apache/gobblin/blob/master/gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/TimeAwareRecursiveCopyableDataset.java#L48), but in templates where this Filebasedsource are used there is no such property so even if we use the recursive property key the value will be empty